Ray.jl/src/runtime.jl

```diff
@@ -155,7 +155,8 @@ end
 initialize_coreworker_driver(args...) = ray_jll.initialize_coreworker_driver(args...)
 
 function submit_task(f::Function, args::Tuple, kwargs::NamedTuple=NamedTuple();
-                     runtime_env::Union{RuntimeEnv,Nothing}=nothing)
+                     runtime_env::Union{RuntimeEnv,Nothing}=nothing,
+                     resources::Dict{String,Float64}=Dict("CPU" => 1.0))
```

Comment:
if we don't set a fallback here does the job error? what happens to the other `options.resources` that aren't set?

Reply:
if you leave it blank, there are no resource requests, which means that ray is then free to over-schedule the CPU.  python uses `CPU: 1` as teh default.  I think there _are_ some concurrency limits on the raylet side (I've seen it spin up more workers than CPUs locally , but not _all_ the tasks get scheduled at once)

Reply:
Do you know where that default is set? From what I've seen it's left undefined

Reply:
not sure, but the logs definitely show them with CPU: 1